stable.txt

```diff
@@ -1 +1 @@
-controller-v0.49.0
+controller-v0.49.0
```

Comment:
until we cut v1, were going to keep this stable?

Reply:
yeap! so people relying on stable.txt file can grab it using scripts.

The change here is due to some extra line that was added in past, but it's still the same file